### ISSUE TITLE
Searching article text

### ISSUE THREAD

**Issue by mpolden** [Ranchero-Software/NetNewsWire#2046]
For some reason I'm unable to search for strings in articles using Cmd+F in the macOS version. 

The menu option `Edit -> Find -> Find...` is grayed out. Is this not supported?

**Comment by brentsimmons** [comment]
Correct. We haven’t implemented Find yet. It’s something we’d like to do, of course.

**Comment by Wevah** [comment]
Interestingly/frustratingly, `WKWebView` declares conformance to `NSTextFinderClient` on 10.15+, but I can’t seem to make it work.

**Comment by Wevah** [comment]
Looks like `WKWebView's` built-in search implementation is private.

**Comment by danielpunkass** [comment]
I implemented this a few years ago, it was merged at the time, but appears to have fallen out of the shipping source tree. If 99797586b11cca3a1f15deaa8c2c019e8216c86b can be cherry-picked back into the source tree I think this could be closed as implemented.
